### PR TITLE
chore/add-action

### PR DESCRIPTION
**Pull request by julianoborba**
Add GitHub Action

### CODE REVIEW

**Review by carloosamarante — DISMISSED** [review]
i don't know if there is something missing, but the yml lgtm

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # X9Containers
 Docker containers to finger pointing pesky breaches from your running images.
 
+It's nothing really new: X9Containers use existing tools to find malwares and vulnerabilities using [Trivy](https://github.com/aquasecurity/trivy) and [ClamAV](https://github.com/Cisco-Talos/clamav) in a [multi-stage building approach](https://docs.docker.com/develop/develop-images/multistage-build/), copying the whole filesystem to be analyzed at each stage - at `distroless.` suffix.
+
+But the analysis can also occur during building, so it can run into the intermediate container, where we hope to find things dynamically - at non `distroless.` suffix.
+
 X9 it is a popular slang among Brazilians wich can be expressed as whistleblower.
```
Comment (pedrokiefer) [inline README.md:8]: `wich` => `which means whistleblower`

File: distroless.high.X9.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+FROM $IMAGE as base
+
+FROM alpine:3.13 as base-stage
+COPY --from=base / ../base-root
+
+FROM base-stage as trivy-stage
+WORKDIR /scans
+COPY --from=aquasec/trivy:latest /usr/local/bin/trivy /usr/local/bin/trivy
+RUN trivy filesystem --ignore-unfixed --severity HIGH,CRITICAL --exit-code 0 --no-progress /base-root | tee image-vulnerabilities-trivy.txt
```
Comment (pedrokiefer) [inline distroless.high.X9.Dockerfile:11]: Why not have severity as an argument?

File: distroless.unknown.X9 copy.Dockerfile [hunk 3]
```diff
@@ -0,0 +1,15 @@
+ARG IMAGE
```
Comment (pedrokiefer) [inline distroless.unknown.X9 copy.Dockerfile:1]: `distroless.unknown.X9 copy.Dockerfile` ? copy?

Comment (julianoborba) [reply distroless.unknown.X9 copy.Dockerfile:1]: OMG the ctrl+c/v curse. I will fix.

File: README.md (hunk 1 above)
Comment (carloosamarante) [reply README.md]: Three witches watch three Swatch watches. Which witch watch which Swatch watch?

Comment (julianoborba) [reply README.md]: @carloosamarante the answer is [Wishiwashi (Pokémon)](https://bulbapedia.bulbagarden.net/wiki/Wishiwashi_(Pok%C3%A9mon)#:~:text=Wishiwashi%20(Japanese%3A%20%E3%83%A8%E3%83%AF%E3%82%B7%20Yowashi),it%20has%20reached%20level%2020.).

File: distroless.high.X9.Dockerfile (hunk 2 above)
Comment (julianoborba) [reply distroless.high.X9.Dockerfile]: Done.

**Review by carloosamarante — APPROVED** [review]

**Review by pedrokiefer — APPROVED** [review]
LGTM

**Pull request merged**